app/services/item-results/enhancers/equivalent-pricings.ts

```diff
@@ -10,10 +10,10 @@ import {ItemResultsEnhancerService, ItemResultsParsedItem} from 'better-trading/
 const CHAOS_IMAGE_URL = 'https://web.poecdn.com/image/Art/2DItems/Currency/CurrencyRerollRare.png';
 const CHAOS_ALT = 'chaos';
 const CHAOS_SLUG = 'chaos-orb';
-const EXALT_IMAGE_URL = 'https://web.poecdn.com/image/Art/2DItems/Currency/CurrencyAddModToRare.png';
-const EXALT_ALT = 'exalt';
-const EXALT_SLUG = 'exalted-orb';
-const EXALT_EQUIVALENCE_THRESHOLD = 0.5;
```

Comment:
about that one, I wonder if it could be a good idea to use generic term for variables and constants. that way, if GGG decide to swap the "main" currency again, we wont have to touch the implementation at all.

thoughts ?